internal/fsutil/create.go

```diff
@@ -16,6 +16,8 @@ type CreateOptions struct {
 	Mode fs.FileMode
 	Data io.Reader
 	Link string
+	// If OverrideMode is true and a directory exists, its mode will be modified.
+	OverrideMode bool
```

Comment:
This is not looking right, for two reasons:

1) We've added a flag to control behavior that was standard until this commit, meaning every single call site for the Create function was the equivalent of OverrideMode: true.   Yet, we're not using this flag _anywhere_ apparently, which means it's false 100% of the time and the behavior has been disabled.  That implies to me that the old behavior has turned irrelevant after the last couple of changes, so we can just drop the behavior altogether.

2) This flag doesn't seem to sit well in the API as a whole. It doesn't make sense for this flag to work only on directories, for example.

GIven (1), what's the situation?  Can we just delete the behavior altogether?

Reply:
Agree with both. I was keeping it for future compatibility but when we need it we can just add it back to the API but deleting unused code is the best code.